src/frontend/src/handler/create_function.rs

```diff
@@ -186,6 +188,41 @@ pub async fn handle_create_function(
                     .into_string(),
             );
         }
+        "deno" => {
+            identifier = function_name.to_string();
+            match (params.using, params.as_) {
+                (None, None) => {
+                    return Err(ErrorCode::InvalidParameterValue(
+                        "Either USING or AS must be specified".into(),
+                    )
+                    .into())
+                }
+                (None, Some(_as)) => body = Some(_as.into_string()),
+                (Some(CreateFunctionUsing::Link(link)), None) => {
+                    let bytes = download_code_from_link(&link).await?;
+                    compressed_binary = Some(zstd::stream::encode_all(bytes.as_slice(), 0)?);
```

Comment:
I don't think javascript source code needs to be compressed and stored in `compressed_binary`. Perhaps storing the plain code in `body` is enough.

Reply:
This is for the cases where you want to load compiled javascript (as for the sse library)

Reply:
But is the compiled bytecode stable across different V8 version? If not, I'm afraid the compiled code won't run on a newer runtime in the future.

Reply:
Got it. That sounds reasonable.